redbot/core/bank.py

```diff
@@ -766,6 +768,10 @@ def _invalid_amount(amount: int) -> bool:
 
     """
     amount = int(amount)
+    max_bal = await get_max_balance(guild)
```

Comment:
> Looking much better, the only other thing is that the payday and starting balance values can be set higher than the max balance if the max balance is changed to lower than either of those values. It isn't really an issue since the balance will be set to where it should be at some point anyway, however it is a small inconsistency. If you can think of a good way to fix it, fix it. If not, don't stress about it.

As discussed in discord
We have 2 options:

1.  Set it the payday amount and initial balance amount  with the `bankset maxbal` command if the old value is higher than the current max

1.  Change the values in  `bankset` and `economyset` help message to the max value if they are higher than the current max value

My issue with 1 is that since this is in the Bank cog it shouldn't edit settings for the Economy Cog, with 2 it's technically False values even though